packages/component-library/components/MenuList/Menu.module.scss

```diff
@@ -13,6 +14,8 @@ $side-padding: 3/4 * $ca-grid;
   border: 1px solid $kz-color-wisteria-200;
   box-shadow: $ca-box-shadow;
   line-height: 40px;
+  border-radius: $kz-border-solid-border-radius;
+  overflow: hidden; /* clip the background colors on child hover */
```

Comment:
Something to watch out for with `overflow: hidden` is that it can potentially clip tooltips and anything else that might appear outside the menu (whether having a tooltip in a menu is a good pattern or not, we definitely have examples of this). I don't _think_ our Kaizen tooltips are absolutely positioned to the top of the document, so I think they'd get clipped (I think they _should_ be absolutely positioned to the top of the doc to avoid these issues, but that that's a separate issue). I could be wrong, but it's worth a check.

One thing I've been thinking about are "integration stories". I need to do this recently to solve z-index issues (the two components were fine by themselves, but not when integrated). Perhaps we could have a story with a tooltip inside one of items to see if there's an issue. If there's an issue, I'd suggest leaving this out and circle back to it later.

Another example of something that could be clipped would sub menus, but that would come down to implementation of that (I don't think this component has ever been used to create fly out sub menus)

Reply:
> integration stories

💯 I can't wait until we have visual regression tests. 

I'm leaning towards removing this and coming back to it later.